about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mindvalley Summit 2021</title>
+    <link href="styles/main.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/5fe0ceda71.js" crossorigin="anonymous"></script>
+    <script src="script.js" defer></script>
+</head>
+
+<body>
+    <header class="toolbar container">
+        <nav class="nav-mobile">
+            <i class="fas fa-bars"></i>
+        </nav>
+        <nav class="nav-desktop">
+            <a class="logo" href="index.html"><img
+                    src="https://a.storyblok.com/f/60579/x/185af974a6/mv-logo-horizontal-colour.svg"
+                    alt="mindvalley logo"></a>
+            <img class="close" src="images/home/close.png" alt="close icon">
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="">CC Campain</a></li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
- [ ] The desktop size of your page should be divided in a top dark bar and a bottom white bar.
- [ ] The top short bar section should have social media links, logout, and language links aligned to the right.<= [This bar is missing]

Reply:
Thank you, I fixed it.

Reply:
:+1: This issue comment is addressed successfully, kudos :muscle: